models/articles-model.js

```diff
@@ -26,3 +26,23 @@ exports.selectArticlesByID = (articles_id) => {
       return data.rows[0];
     });
 };
+
+exports.updateArticleByID = (inc_votes, articles_id) => {
+  if (inc_votes === 0 || inc_votes < -1 || inc_votes > 1 ) {
```

Comment:
We should actually be able to increment our votes by more than 1, and also use this to decrement the votes. So ideally we want to remove this check.